### PR TITLE
Infineon p6 all clusters update

### PR DESCRIPTION
**Pull request by keithmwheeler**
#### Problem
Fixed commissioning issue with all clusters; updated network commissioning wifi driver with needed lock/unlock sequences.

#### Change overview
Updated P6 all clusters and all clusters minimal apptask to support commissioning only on endpoint 0. Added necessary lock/unlock calls to P6 network commissioning wifi driver.

#### Testing
 Tested by commissioning both all clusters and all clusters minimal on P6 hardware.

### CODE REVIEW

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]
This should not be making changes to the EFR32 SDK or the ZAP repo... and probably not to mbedtls.

File: examples/chip-tool/commands/interactive/InteractiveCommands.cpp [hunk 1]
```diff
@@ -102,9 +104,10 @@ bool InteractiveStartCommand::ParseCommand(char * command)
     char * args[kInteractiveModeArgumentsMaxLength];
     args[0]       = kInteractiveModeName;
     int argsCount = 1;
+    std::string arg;
```
Comment (bzbarsky-apple) [inline examples/chip-tool/commands/interactive/InteractiveCommands.cpp:107]: These changes look unrelated...

File: src/credentials/CHIPCert.cpp [hunk 2]
```diff
@@ -1013,7 +1013,7 @@ DLL_EXPORT CHIP_ERROR ASN1ToChipEpochTime(const chip::ASN1::ASN1UniversalTime &
     CHIP_ERROR err = CHIP_NO_ERROR;
 
     // X.509/RFC5280 defines the special time 99991231235959Z to mean 'no well-defined expiration date'.
-    // In CHIP certificate it is represented as a CHIP Epoch UTC time value of 0 sec (2020-01-01 00:00:00 UTC).
+    // In CHIP certificate it is represented as a CHIP Epoch UTC time value of 0 sec (2000-01-01 00:00:00 UTC).
```
Comment (bzbarsky-apple) [inline src/credentials/CHIPCert.cpp:1016]: These also look unrelated.

Comment (keithmwheeler) [reply src/credentials/CHIPCert.cpp]: Should be corrected in current commit.

**Review by bzbarsky-apple — APPROVED** [review]

File: examples/chip-tool/commands/interactive/InteractiveCommands.cpp (hunk 1 above)
Comment (keithmwheeler) [reply examples/chip-tool/commands/interactive/InteractiveCommands.cpp]: Should be corrected in latest commit.

**Review by bzbarsky-apple — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**